### ISSUE TITLE
I don't know??

### ISSUE THREAD

**Issue by Delaspaingg** [thatcherclough/AttendanceBot#4]
Hi, finally I don't know how to used... Can u explain to me how I used? pls

**Comment by thatcherclough** [comment]
Yes. So this program is used to fill out google form based attendance. Does your school take attendance through google forms? If so, if you save the google form page (through File -> Save page) I can help you tailor the program to your needs!
